std/types/react-dom/README.md

```diff
@@ -11,3 +11,21 @@ Facebook's React library.
 // @deno-types="https://deno.land/std/types/react-dom/v16.13.1/react-dom.d.ts"
 import ReactDOM from "https://cdn.pika.dev/@pika/react-dom@v16.13.1";
 ```
+
+```typescript
+// @deno-types="https://deno.land/std/types/react-dom/v16.9.0/server/react-dom-server.d.ts"
```

Comment:
I think the original file structure should be retained

the file should be `react-dom/server.d.ts`, not `react-dom-server.d.ts`

Reply:
Minor change. Will do it in a while